lnrpc/lightning.proto

```diff
@@ -4029,10 +4044,10 @@ message ForwardingEvent {
     uint64 amt_out = 6;
 
     // The total fee (in satoshis) that this payment circuit carried.
-    uint64 fee = 7;
+    int64 fee = 7;
```

Comment:
I'm not sure how breaking this change is for clients already subscribing to forwarding events? Could be the number is encoded differently, so a completely different number is deserialized on the client side if they expect this to be a uint64?

Reply:
Yes, I considered that too. My thinking was that these values can only turn negative if you actually use negative inbound fees. So in no case would this affect node operators that don't use negative fees.

An alternative is to a new signed field, but what to do then with the old field? Round up to zero always?

Reply:
Updated PR to remain backwards compatible here. Existing fields are reporting zero if the fee value is negative.